src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -361,11 +360,11 @@ export class RunParametersAction extends TooltipFromLabelAction {
 	 * TODO - Call Extensibility API for ShowNotebook
 	 * (showNotebookDocument to be utilized in Notebook Service)
 	**/
-	public async openParameterizedNotebook(uri: URI): Promise<void> {
+	public async openParameterizedNotebook(uri: URI, uriParams: string): Promise<void> {
```

Comment:
This code shouldn't be necessary now that you're handling the URI stuff on the service side right?

Reply:
Correct!